contributing.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Broyster
+
+## Working with the code
+
+Make sure you have [Yarn](https://yarnpkg.com) installed.
+
+### Development
+
+The project requires little start-up efforts and verification of most enhancement work.
```

Comment:
I am still not sure what you are trying to say here. What is verification? What is enhancement? I don't have a lot of experience in testing, maybe others will understand but try to explain it to me like I am a golden retriever :)